src/V1/Api/Dumper.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neighborhoods\Buphalo\V1\Api;
+
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Yaml;
+use Symfony\Component\Serializer;
+
+class Dumper implements DumperInterface
+{
+    private const DUMP_FORMAT = Serializer\Encoder\YamlEncoder::FORMAT;
+    private const YAML_INDENTATION = 2;
+
+    /** @var Serializer\Serializer */
+    private $serializer;
+
+    /** @var Filesystem */
+    private $filesystem;
+
+    public function __construct()
+    {
+        // If using PHP 7.4+ style setters, we could specify an ObjectNormalizer here
+        // This would allow us to pass `AbstractOjbectNormalizer::SKIP_NULL_VALUES = true` below.
+        // However, that Normalizer actually detects and uses getters, but doesn't know about hassers,
```

Comment:
Weird. If it's using `jsonSerialize()`, why would it care about getters?

Reply:
The `ObjectNormalizer` is what will let it skip null values, which is separate from the jsonSerialize thing. Though we could probably also create our own SkipNullValues normalizer if we really wanted to do this. And that one would probably be inserted after the JsonSerializer one.